docs/instrument-traces/go-app.md

```diff
@@ -50,114 +40,92 @@ import (
     "go.opentelemetry.io/otel/sdk/resource"
     sdktrace "go.opentelemetry.io/otel/sdk/trace"
 )
-
 ```
-To configure the application to send data we have to create a function to initialize Opentelemetry. Update the code in `main.go` file by adding the code given below:
-
-```
-package main
-
-import (
-    "context"
-    "github.com/gin-gonic/gin"
-    "github.com/siglens/bookstore-app/controllers"
-    "github.com/siglens/bookstore-app/models"
-    "go.opentelemetry.io/contrib/instrumentation/github.com/gin-gonic/gin/otelgin"
-    "go.opentelemetry.io/otel"
-    "go.opentelemetry.io/otel/attribute"
-    "go.opentelemetry.io/otel/exporters/otlp/otlptrace"
-    "go.opentelemetry.io/otel/exporters/otlp/otlptrace/otlptracehttp"
-    "go.opentelemetry.io/otel/sdk/resource"
-    sdktrace "go.opentelemetry.io/otel/sdk/trace"
-    "log"
-    "os"
-)
 
+2. Setup an `initTracing()` function in your `main.go`:
+```golang
 var (
-    serviceName = os.Getenv("SERVICE_NAME")
+	serviceName = os.Getenv("SERVICE_NAME")
 )
 
-func initTracer() func(context.Context) error {
-
-    exporter, err := otlptrace.New(
-        context.Background(),
-        otlptracehttp.NewClient(
-            otlptracehttp.WithURLPath("/otlp/v1/traces"),
-            otlptracehttp.WithInsecure(),
-        ),
-    )
-
-    if err != nil {
-        log.Fatalf("Failed to create exporter: %v", err)
-    }
-    resources, err := resource.New(
-        context.Background(),
-        resource.WithAttributes(
-            attribute.String("service.name", serviceName),
-            attribute.String("library.language", "go"),
-        ),
-    )
-    if err != nil {
-        log.Fatalf("Could not set resources: %v", err)
-    }
-
-    otel.SetTracerProvider(
-        sdktrace.NewTracerProvider(
-            sdktrace.WithSampler(sdktrace.AlwaysSample()),
-            sdktrace.WithBatcher(exporter),
-            sdktrace.WithResource(resources),
-        ),
-    )
-    return exporter.Shutdown
+func initTracing() func(context.Context) error {
+
+	exporter, err := otlptrace.New(
+		context.Background(),
+		otlptracehttp.NewClient(
+			otlptracehttp.WithInsecure(), // TODO: remove in production to use https, not http
+			otlptracehttp.WithEndpoint("localhost:8081"),
```

Comment:
nit: we should remove this endpoint from here and add it as a env var when runing the sample go app